src/edu/asu/plp/tool/prototype/Main.java

```diff
@@ -413,45 +463,843 @@ private ProjectExplorerTree createProjectTree()
 	private Parent createToolbar()
 	{
 		HBox toolbar = new HBox();
-		toolbar.setPadding(new Insets(0, 0, 0, 5));
+		toolbar.setPadding(new Insets(1.5, 0, 1, 5));
 		toolbar.setSpacing(5);
 		ObservableList<Node> buttons = toolbar.getChildren();
 		
 		EventHandler<MouseEvent> listener;
 		Node button;
 		
+		DropShadow lBlueShadow = new DropShadow();
+		lBlueShadow.setColor(Color.LIGHTBLUE);
+		DropShadow dBlueShadow = new DropShadow();
+		dBlueShadow.setColor(Color.DARKBLUE);
+		
 		// TODO: replace event handlers with actual content
+		Node projectButton = new ImageView("toolbar_new.png");
+		projectButton.addEventHandler(MouseEvent.MOUSE_ENTERED,
+				new EventHandler<MouseEvent>() {
+					@Override
+					public void handle(MouseEvent e)
+					{
+						projectButton.setEffect(lBlueShadow);
+					}
+				});
```

Comment:
This would be cleaner as a functional argument. Example:
```
projectButton.addEventHandler((event) -> projectButton.setEffect(lBlueShadow));
```
or
```
projectButton.addEventHandler(this::onNewProjectClicked);
```
with the following method:
```
public void onNewProjectClicked(MouseEvent event)
{
    projectButton.setEffect(lBlueShadow);
}
```